packages/reports/src/Containers/open-positions.jsx

```diff
@@ -342,26 +380,50 @@ const OpenPositions = ({
     if (error) return <p>{error}</p>;
 
     const is_multiplier_selected = has_multiplier_contract && active_index === 1;
+    const is_accumulator_selected =
+        (has_accumulator_contract && !has_multiplier_contract && active_index === 1) ||
+        (has_accumulator_contract && active_index === 2);
     const active_positions_filtered = active_positions?.filter(p => {
         if (p.contract_info) {
-            return is_multiplier_selected
-                ? isMultiplierContract(p.contract_info.contract_type)
-                : !isMultiplierContract(p.contract_info.contract_type);
+            if (is_multiplier_selected) {
+                return isMultiplierContract(p.contract_info.contract_type);
+            } else if (is_accumulator_selected) {
+                return isAccumulatorContract(p.contract_info.contract_type);
+            }
+            return (
+                !isMultiplierContract(p.contract_info.contract_type) &&
+                !isAccumulatorContract(p.contract_info.contract_type)
+            );
```

Comment:
@NikitK-deriv unfortunately, it will not work, not only because ternary operations are not assigned to a variable, but mainly because it's not enough here to just assign ternary operation to variables and return result once. We have to return the result every time we check.
This is what's going on here: When is_multiplier_selected, we check if isMultiplierContract(p.contract_info.contract_type) is true. When is_accumulator_selected, we check if isAccumulatorContract(p.contract_info.contract_type) is true. If none of them is selected, we return true only if it's neither multiplier contract, nor accumulator contract.